test-framework/common/src/main/java/io/quarkus/test/common/NativeImageLauncher.java

```diff
@@ -52,6 +52,7 @@ public void start() throws IOException {
         List<String> args = new ArrayList<>();
         args.add(path);
         args.add("-Dquarkus.http.port=" + port);
+        args.add("-Dquarkus-profile=test");
```

Comment:
@machi1990 OK, so I was wrong, could you change that back :)

And promote your PR so that we can include it?

Thanks!

Reply:
It is ready for review now.